src/OpenTelemetry.Extensions/Logs/OpenTelemetryLoggingExtensions.cs

```diff
@@ -34,4 +36,45 @@ public static OpenTelemetryLoggerOptions AttachLogsToActivityEvent(
         return loggerOptions.AddProcessor(new ActivityEventAttachingLogProcessor(options));
 #pragma warning restore CA2000 // Dispose objects before losing scope
     }
+
+    /// <summary>
+    /// Adds the <see cref="BaggageLogRecordProcessor"/> to the <see cref="OpenTelemetryLoggerOptions"/>.
```

Comment:
please mention what does this do..specifically stating it adds *every* Baggage entry to every LogRecord.